DesignPatterns/DesignPatterns.ThreadSafetySingleton/ThreadSafetySingleton.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace DesignPatterns.ThreadSafetySingleton
+{
+    public class ThreadSafetySingleton
+    {
+        private static ThreadSafetySingleton _instance;
+        private static readonly object _locker;
```

Comment:
In order to use object for lock functionality, it should be initialized.
private static readonly object _locker = new object();